src/cryptoadvance/specter/server_endpoints/devices.py

```diff
@@ -174,8 +180,8 @@ def new_device_keys(device_type):
                 )
 
     return render_template(
-        "device/new_device/new_device_keys.jinja",
-        device_class=get_device_class(device_type),
+        template,
+        device_class=device_class,
```

Comment:
It would be good if the form fails and refreshed, that the previously filled out fields are in there.    
```
        device_name=device_name,
        xpub=xpub,
        derivation_path=derivation_path,
        root_fingerprint=root_fingerprint,
```

Reply:
I don't think this is so important, since we are dealing with only 3 input fields here. Having said that, feel free to implement.

Reply:
Ok, I'll do that :-)

Reply:
That's done, no?